### ISSUE TITLE
List blobs by prefix

### ISSUE THREAD

**Issue by g4s8** [artipie/files-adapter#77]
Let's add a feature to list blobs in repository by prefix (simulating `ls` for directory) E.g. if we have repo blobs `foo/bar/file1`, `foo/bar/file2` and `foo/bar/file3`, then the request `GET /foo/bar` should list  all blobs in response:
```
200 OK
Content-Type: text/plain
Content-Length:

foo/bar/file1
foo/bar/file2
foo/bar/file3
```

The trailing slash in request is optional (`GET /foo/bar` is equal to `GET /foo/bar/`).
We should perform [content negotiation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation) by `Accept` headers. Let's support these types:
 - `text/plain` - just a list of blobs separated by `\n`
 - `text/html` - simple markdown with `<ul>` and `<li>` + `<a>` tags:
```html
         <html>
           <body>
             <ul>
               <li><a href="base-uri/foo/bar/file1">foo/bar/file1</a></li>
               <!-- same for file2 and file3 -->
             </ul>
           </body>
         <html>
```
 - `application/json` - simple JSON array with blob names, full URIs and maybe additional metadata:
```json
    [
      {
        "key": "foo/bar/file1",
        "uri": "base-uri/foo/bar/file1",
      },
      // same for file2 and file3
    ]
```

(**to performer**: please pay attention, this task is estimated for 30 minutes, `@todo` or subtasks could be used for decomposing).

**Comment by kirill-h5i** [comment]
@g4s8 reassign

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#90](https://github.com/artipie/files-adapter/issues/90), [#91](https://github.com/artipie/files-adapter/issues/91) are still not solved.

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#91](https://github.com/artipie/files-adapter/issues/91) is still not solved; solved: [#90](https://github.com/artipie/files-adapter/issues/90).

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#94](https://github.com/artipie/files-adapter/issues/94), [#95](https://github.com/artipie/files-adapter/issues/95) are still not solved; solved: [#90](https://github.com/artipie/files-adapter/issues/90), [#91](https://github.com/artipie/files-adapter/issues/91).

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#95](https://github.com/artipie/files-adapter/issues/95) is still not solved; solved: [#90](https://github.com/artipie/files-adapter/issues/90), [#91](https://github.com/artipie/files-adapter/issues/91), [#94](https://github.com/artipie/files-adapter/issues/94).
